src/main/java/frc/robot/controls/GameControllerDriverConfig.java

```diff
@@ -32,6 +37,16 @@ public void configureControls() {
     )));
     kDriver.get(Button.X).whileTrue(new SetFormationX(super.getDrivetrain()));
 
+    // Moves to the selected scoring position using Path Planner
+    kDriver.get(kDriver.LEFT_TRIGGER_BUTTON).whileTrue(new GoToNode(m_operator, getDrivetrain()));
+    // Moves to the selected scoring position using the PID
+    kDriver.get(Button.LB).whileTrue(new GoToNodePID(m_operator, getDrivetrain()));
+
+    // Moves to the shelf using Path Planner
+    kDriver.get(kDriver.RIGHT_TRIGGER_BUTTON).whileTrue(new GoToShelf(getDrivetrain()));
+    // Moves to the shelf using the PID
+    kDriver.get(Button.RB).whileTrue(new GoToShelfPID(getDrivetrain()));
```

Comment:
I think there are already buttons on the triggers for slowmode and angle align, for now I would rebind those to different buttons

Reply:
Where are those controls? I don't see them in the game controller class. These buttons are just for testing, but we probably will only use PathPlanner later and not the PID, so all alignment controls can be moved to the left side later.